pkg/transformer/functions.go

```diff
@@ -41,6 +47,64 @@ func GenericFuncMap() map[string]interface{} {
 	return gfm
 }
 
+// Get from https://github.com/Masterminds/sprig/blob/master/defaults.go
+//
+// Checks whether `given` is set, and returns default if not set.
+func defaultFunc(d interface{}, given ...interface{}) interface{} {
+	if empty(given) || empty(given[0]) {
+		return d
+	}
+
+	return given[0]
+}
+
+// empty returns true if the given value has the zero value for its type.
+func empty(given interface{}) bool {
+	g := reflect.ValueOf(given)
+	if !g.IsValid() {
+		return true
+	}
+
+	switch g.Kind() { //nolint:exhaustive
+	default:
+		return g.IsNil()
+	case reflect.Array, reflect.Slice, reflect.Map, reflect.String:
+		return g.Len() == 0
+	case reflect.Bool:
+		return !g.Bool()
+	case reflect.Complex64, reflect.Complex128:
+		return g.Complex() == 0
+	case reflect.Int, reflect.Int8, reflect.Int16, reflect.Int32, reflect.Int64:
+		return g.Int() == 0
+	case reflect.Uint, reflect.Uint8, reflect.Uint16, reflect.Uint32, reflect.Uint64, reflect.Uintptr:
+		return g.Uint() == 0
+	case reflect.Float32, reflect.Float64:
+		return g.Float() == 0
+	case reflect.Struct:
+		return false
+	}
```

Comment:
```suggestion
	return !g.IsValid() || g.IsZero()
```

Perhaps?

Reply:
Yeah it was copy from https://github.com/Masterminds/sprig/blob/master/defaults.go

Thanks! I've applied the changes